src/middlewared/middlewared/plugins/disk.py

```diff
@@ -676,10 +688,12 @@ def check_clean(self, disk):
     @private
     async def sed_unlock_all(self):
         advconfig = await self.middleware.call('system.advanced.config')
-        disks = await self.middleware.call('disk.query')
+        disks = await self.query([], {'extra': {'passwords': True}})
```

Comment:
I personally prefer to keep middleware.call in these situations, because if we change query to being a method this wont silently break